src/lms7002m/LMS7002M.cpp

```diff
@@ -33,7 +33,7 @@ using namespace lime;
 #include "MCU_BD.h"
 
 float_type LMS7002M::gVCO_frequency_table[3][2] = { { 3800e6, 5222e6 }, { 4961e6, 6754e6 }, {6306e6, 7714e6} };
-float_type LMS7002M::gCGEN_VCO_frequencies[2] = {1930e6, 2940e6};
+float_type LMS7002M::gCGEN_VCO_frequencies[2] = {2000e6, 2700e6};
```

Comment:
Is this wanted?

Reply:
Yes, see: https://github.com/xtrx-sdr/liblms7002m/blob/master/liblms7002m.h#L253

Reply:
Actually, I'm not so sure anymore. It seems to have changed in the past: https://github.com/myriadrf/LimeSuite/commit/ae3bd0fdf5de6826f79dcbd42e5f6287c3f67286
I changed it back to the original value and the VCO can still be tuned.